src/languages/php.js

```diff
@@ -431,11 +430,59 @@ export default function(hljs) {
   };
   PARAMS_MODE.contains.push(FUNCTION_INVOKE);
 
+  const ATTRIBUTES = {
+    begin: regex.concat(/^\s*#\[\s*\\?/, PASCAL_CASE_CLASS_NAME_RE),
```

Comment:
This should not match whitespace... our HTML scope output should be clean:


```html
<!-- bad -->
<span class="highlight">             keyword</span>
<!-- good -->
<span class="highlight">keyword</span>
```

Could this be confused for something else, what if we just matched starting at the `#[`?

Reply:
So I should highlight this with two meta tags?
```    
#[
         Assert\Collection(
```

Reply:
```html
<span>#[
         Assert\Collection(</span>
```

Is acceptable.  This is what I"m objecting to:

```html
<span>
<!-- tons of whitespace -->

#[
         Assert\Collection(</span>
```

Reply:
> No.
> 
> ```
> <span>#[
>          Assert\Collection(</span>
> ```
> 
> Is acceptable. This is what I'm objecting to:
> 
> ```
> <span>
> <!-- tons of whitespace before actual content -->
> 
> #[
>          Assert\Collection(</span>
> ```

Fixed